### ISSUE TITLE
Scam, mallware etc.

### ISSUE THREAD

**Issue by WarezzK** [phw198/OutlookGoogleCalendarSync#824]
After i use your plugin - do you tell me WTF is this shit? You think you're cool?

![PrintScreen_01_09](https://user-images.githubusercontent.com/9317317/64074387-e16c7980-ccaa-11e9-9fab-3735992c4da6.png)

**Comment by phw198** [comment]
Thanks for your constructive feedback. Given OGCS isn't a plugin I'm not even sure you're posting to the right project....and with such limited information it would be virtually impossible to diagnose.

**Comment by fareloz** [comment]
@phw198 
It also happened to me, I sync my work outlook calendar to google one-way.
I think you have added this ads to appear in calendars of free users.
Please, remove them, or add a note for free version.

**Comment by phw198** [comment]
I have no idea what you've talking about. There are no ads from OGCS, though I have heard people reporting Google themselves adding ads into calendars...

**Comment by fareloz** [comment]
@phw198 
Might be, I can't prove something here, all I can say - it happened the same day I installed calendar sync.

**Comment by phw198** [comment]
@fareloz What happened exactly? Do you have a screenshot of the problem or can provide your OGcalsync.log file for the sync you think caused it?

**Comment by fareloz** [comment]
@phw198 
Exactly, after using the app, the entry from screenshot above appeared in my google calendar (regarding win IPhone spam).
Sorry, but I can't provide any logs, because I deleted the app and no longer use it.
Happened to be here because I was googling for an alternative.

**Comment by phw198** [comment]
I knew I'd read something about Google Calendar spam before - and yes, it's nothing to do with OGCS.

See this [reddit post](https://www.reddit.com/r/Android/comments/cuh6aw/your_iphone_xs_max_is_ready_for_pickup_calendar/exuzxi2/) for more information. I'll do some further research and maybe write a Wiki entry for it.

**Comment by fareloz** [comment]
That's exactly the issue I had.
Writing a wiki entry is a good idea. Otherwise there might be more aggressive topics like this in the future.

**Comment by WarezzK** [comment]
First i must appologize for my offensive post. Posting was be faster than thinking. I hope this will be some day resolved. Sorry man. I will continue using your app but when Outlook ask he need a permission 3rd party app i choose deny. It will be temporary solution.

**Comment by phw198** [comment]
@WarezzK Well thanks for returning and apologising :+1: :handshake: 
Worth remembering it's extremely unlikely an open source project is going to inject rubbish into its user's applications - that'd be one way to ensure no one uses it!

However, you say you've denied OGCS accessing Outlook? That sounds like you need to adjust your security settings/AV definitions - [see here for details](https://github.com/phw198/OutlookGoogleCalendarSync/wiki/FAQs---Outlook-Security#why-am-i-getting-outlook-security-pop-ups). OGCS won't be able to work if you deny it access to Outlook.

**Comment by exmole** [comment]
Sounds to me like your software is retrieving **ALL** google mail from the users' mailbox, and Google automatically filters out spam, but the spam mail that the user has never seen is being retrieved and made visible, misleading the user into thinking that your software has added unwanted mail.

**Comment by phw198** [comment]
@exmole No I don't think you've understood the reddit post properly and OGCS most certainly does **not scan any email** - either in Outlook or Gmail.

The issue is entirely of Google's making. _Google_ scan emails for events to put in the recipient's calendar - but they weren't excluding the spam folder, so spammers were sending invites that Google would helpfully automatically add to the calendar.

Absolutely nothing to do with OGCS aside from the fact it will sync these spams events to Outlook if they are showing in Google calendar.
